README.md

```diff
@@ -79,7 +81,7 @@ Consul has support for versioning (`ModifyIndex`) which allows us to change the
 
 Will be github issues once we move to thebeatapp organization.
 
-- consul client timeouts should be set
+- create a example service in the examples folder (d.baltas)
```

Comment:
Fix typo `create a(n) example...` or better change line to `create examples folder with a service implementation`